ui/README.md

```diff
@@ -7,16 +7,16 @@ The official Nomad UI.
 This is an [ember.js](https://emberjs.com/) project, and you will need the following tools installed on your computer.
 
 - [The current Node.js LTS release](https://nodejs.org/)
-- [Yarn](https://yarnpkg.com)
 - [Ember CLI](https://ember-cli.com/)
 
 ## Installation
 
 The Nomad UI gets cloned along with the rest of Nomad. To install dependencies, do the following from the root of the Nomad project:
 
-```
-$ cd ui
-$ yarn
+```shell
+corepack enable
```

Comment:
I think we need to add this `corepack` thing to the list of dependencies above, right?